.github/workflows/interchaintest.yaml

```diff
@@ -28,14 +24,14 @@ jobs:
           file: Dockerfile
```

Comment:
the build and push action is what does the actual building.  buildx and qemu are for multiplatform only.